main.c

```diff
@@ -0,0 +1,51 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+/* *
+ * @brief program in C to find the largest element using Dynamic Memory Allocation
+ * */
+int main(void)
+{
+  int lenght = 0;
+
+  printf("input lenght");
+  scanf("%d", &lenght);
+  printf("lenght - %d  \n", lenght);
+
+  int array[lenght];
+  int *ptr;
+
+  if((ptr = malloc(sizeof(array))) != NULL)

```

Comment:
If you using the malloc you need also free memory when memory will be unused. In you case you losing memory, in other words you have memory leak in this program‼️‼️‼️